google-cloud-trace/lib/google/cloud/trace/v1/trace_service_api.rb

```diff
@@ -0,0 +1,307 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from the file
+# https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudtrace/v1/trace.proto,
+# and updates to that file get reflected here through a refresh process.
+# For the short term, the refresh process will only be runnable by Google
+# engineers.
+#
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+
+require "json"
+require "pathname"
+
+require "google/gax"
+require "google/devtools/cloudtrace/v1/trace_pb"
+
+module Google
+  module Cloud
+    module Trace
+      module V1
+        # This file describes an API for collecting and viewing traces and spans
+        # within a trace.  A Trace is a collection of spans corresponding to a single
+        # operation or set of operations for an application. A span is an individual
+        # timed event which forms a node of the trace tree. Spans for a single trace
+        # may span multiple services.
+        #
+        # @!attribute [r] trace_service_stub
+        #   @return [Google::Devtools::Cloudtrace::V1::TraceService::Stub]
+        class TraceServiceApi
+          attr_reader :trace_service_stub
+
+          # The default address of the service.
+          SERVICE_ADDRESS = "cloudtrace.googleapis.com".freeze
+
+          # The default port of the service.
+          DEFAULT_SERVICE_PORT = 443
+
+          CODE_GEN_NAME_VERSION = "gapic/0.1.0".freeze
+
+          DEFAULT_TIMEOUT = 30
```

Comment:
Any reason this value does not have a comment whereas SERVICE_ADDRESS and DEFAULT_SERVICE_PORT do?